controllers/neutronapi_controller.go

```diff
@@ -279,30 +317,49 @@ func (r *NeutronAPIReconciler) reconcileInit(
 		}
 		r.Log.Info(fmt.Sprintf("Job %s hash added - %s", jobDef.Name, instance.Status.Hash[neutronv1beta1.DbSyncHash]))
 	}
+	instance.Status.Conditions.MarkTrue(condition.DBSyncReadyCondition, condition.DBSyncReadyMessage)
 
 	// run Neutron db sync - end
 
 	//
 	// expose the service (create service, route and return the created endpoint URLs)
 	//
-	var ports = map[common.Endpoint]int32{
-		common.EndpointAdmin:    neutronapi.NeutronAdminPort,
-		common.EndpointPublic:   neutronapi.NeutronPublicPort,
-		common.EndpointInternal: neutronapi.NeutronInternalPort,
+	var keystonePorts = map[endpoint.Endpoint]endpoint.EndpointData{
```

Comment:
with [1], which fixed a linter error, from latest lib-common this will be endpoint.Data

[1] https://github.com/openstack-k8s-operators/lib-common/pull/41/commits/752e673187cdebd192328a2c518943a55a429ca2

Reply:
I must have done `go get -u` against `lib-common` just before that change was merged.  I'll get the latest and fix this.